### PR TITLE
Fix #2823: Profile list activity label

### LINKED ISSUE #2823: [A11y] Add label for ProfileListActivity
## Task
Introduce label for `ProfileListActivity` as `Profile List`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by pravesh00**
## Explanation

Fixes #2823: Profile list activity label added

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by pravesh00** [comment]
PTAL @rt4914 @anandwana001 @Victor-Titan

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -283,7 +283,7 @@
   <string name="home_activity_back_dialog_cancel">Cancel</string>
   <string name="home_activity_back_dialog_exit">Exit</string>
   <!-- ProfileListActivity -->
-  <string name="profile_list_activity_title">Profiles</string>
+  <string name="profile_list_activity_title">Profile List</string>
```
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Lets keep it as before only. Keep it as `Profiles` only.

File: app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListActivityTest.kt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  @Inject
+  lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+  }
+
+  @Test
+  fun testFaqListActivity_hasCorrectActivityLabel() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListActivityTest.kt]: Rename this test

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested nit change. Otherwise looks good.

File: app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListActivityTest.kt (hunk 2 above)
Comment (pravesh00) [reply app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListActivityTest.kt]: PTAL @rt4914 I have made the suggested changes

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**